### PR TITLE
Update WHITEPAPER.md

### PR DESCRIPTION
**Pull request by huangsuyu**

### CODE REVIEW

File: WHITEPAPER.md [hunk 1]
```diff
@@ -181,15 +181,15 @@ The architecture of cross-chain communication is as in the above diagram. To acc
...
 
 1. BC blocking status will be synced to the light client contracts on BSC from time to time, via     block header and pre-commits, for the below information:
     * block and app hash of BC that are signed by validators
     * current validatorset, and validator set update
 
 2. the key-value from the blockchain state will be verified based on the Merkle     Proof and information from above #1.
 
-After confirming the key-value is accurate and trustful, the light client contracts will execute the actions corresponding to the cross-chain packages. Some examples of such packages that can be created for BC-to-BSC are:
+After confirming the key-value is accurate and trustful, the build-in system contracts contracts will execute the actions corresponding to the cross-chain packages. Some examples of such packages that can be created for BC-to-BSC are:
```
Comment (HaoyangLiu) [inline WHITEPAPER.md:192]: > the build-in system contracts contracts

two contracts

File: WHITEPAPER.md [hunk 2]
```diff
@@ -181,15 +181,15 @@ The architecture of cross-chain communication is as in the above diagram. To acc
...
 
-BC-to-BSC communication will be verified in an “**on-chain light client**” implemented via BSC **Smart Contracts** (some of them may be **“pre-compiled”**). After some transactions and state change happen on BC, if a transaction is defined to trigger cross-chain communication, the **Relayers** will create and pass the Cross-chain “**package”** message and submit onto BSC as data into the “light client contracts”. The light client will verify the package and execute the transactions if it passes the verification. The verification will be guaranteed with the below design:
+BC-to-BSC communication will be verified in an “**on-chain light client**” implemented via BSC **Smart Contracts** (some of them may be **“pre-compiled”**). After some transactions and state change happen on BC, if a transaction is defined to trigger cross-chain communication,the Cross-chain “**package**” message will be created and **BSC Relayers** will pass and submit them onto BSC as data into the “build-in system contracts. The build-in system contracts will verify the package and execute the transactions if it passes the verification. The verification will be guaranteed with the below design:
```
Comment (HaoyangLiu) [inline WHITEPAPER.md:184]: > “build-in system contracts

missing "

Comment (huangsuyu) [reply WHITEPAPER.md]: fixed

**Comment by Shednandezstore** [comment]
****Seen the reaction of this chains []()

**Review by flexibrah — APPROVED** [review]

**Comment by Ramezaan7** [comment]
master

**Pull request merged**